recipe-demo-core/Models/RecipeManager.cs

```diff
@@ -85,7 +85,7 @@ public static Recipe GetRecipeRandom()
             double randomNumber = Global.Singleton.Random.NextDouble() / crazyMultiplier;
 
             // DEMO: Dobule visualizer
-            index = (int)(randomNumber * recipes.Count * crazyMultiplier);
+            index = (int)(randomNumber * recipes.Count * crazyMultiplier * 2);
```

Comment:
**Wizard Bot**

When deployed this change is likely to cause an exception in 2% of cases when deployed to the *MyApp* environment.

[Learn How to Fix This]()